package/src/swagger_server/controllers/health.py

```diff
@@ -0,0 +1,42 @@
+from os import environ as env
+
+import pymongo
+
+def _health_check():
+    try:
+        client = pymongo.MongoClient(
+            'mongodb://mongodb:27017/', 
+            username=env.get('MONGO_USERNAME'), 
+            password=env.get('MONGO_PASSWORD'),
+            serverSelectionTimeoutMS=1
+        )
+        client.server_info()
+        return {
+            "status": "ok",
+            "info": {
+                "mongo-query": {
+                    "status": "up"
+                }
+            },
+            "error": {},
+            "details": {
+                "mongo-query": {
+                    "status": "up"
+                }
+            }
+        }
+    except pymongo.errors.ServerSelectionTimeoutError as err:
```

Comment:
Ideally this should return with a HTTP status code of 503 along with the JSON body if the health check fails.

Reply:
Ah, ok. Must have not seen that in the docs on terminus. I tried to remodel what they are doing.